src/main/java/org/graylog/integrations/notifications/types/SlackEventNotificationConfig.java

```diff
@@ -41,9 +42,15 @@
 @JsonDeserialize(builder = SlackEventNotificationConfig.Builder.class)
 public abstract class SlackEventNotificationConfig implements EventNotificationConfig {
 
-	private final String regex = "https:\\/\\/hooks.slack.com\\/services\\/";
-	private final Pattern pattern = Pattern.compile(regex, Pattern.MULTILINE);
-	public static final String TYPE_NAME = "slack-notification-v1";
+	private final  String regex = "https:\\/\\/hooks.slack.com\\/services\\/";
+	private final  Pattern pattern = Pattern.compile(regex, Pattern.MULTILINE);
+	public static  final String TYPE_NAME = "slack-notification-v1";
```

Comment:
The indentation changes here are unrelated. What's the reason for them? We usually just use IntelliJ's standard formatting. (we will soon have our custom style config incl. auto-formatting)